docs/source/user-guide/tutorials/building-conda-packages.rst

```diff
@@ -277,6 +276,20 @@ should be installed when the package is installed.
    important to use the same NumPy version at runtime that was used
    at build time.
 
+
+Create a conda_build_config.yaml
+--------------------------------
+
+Conda build will default to using the VS 2015 compilers unless specified
+otherwise in a conda_build_config.yaml. Create a conda_build_config.yaml
```

Comment:
It might be better if we just update conda build to use vs2017 by default. That's probably sane now and it'll greatly simplify this tutorial.